### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by james-aguirre**
Kapture 2023-04-11 at 10 51 37

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  height: 30px;
+  border-radius: 10px;
+  margin-left: 600px;
+  margin-top: 2%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
```
Comment (shawnkost) [inline css/styles.css:25]: This is already defined in `layout.css`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const $form = document.querySelector('#form');
+
+document.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const messageData = {};
+  messageData.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  messageData.title = $form.title.value;
+  messageData.img = $form.img.value;
+  messageData.notes = $form.notes.value;
```
Comment (shawnkost) [inline js/main.js:17]: You should be accessing the values from the `event` object

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here James!

**Comment by james-aguirre** [comment]
Kapture 2023-04-11 at 11 42 56

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**